consumers/email-consumer/src/email_consumer/email_consumer.py

```
@@ -0,0 +1,46 @@
+                data={
+                    "from": f"Codesirius <no-reply@{os.environ.get('MAILGUN_DOMAIN')}>",
+                    "to": email,
+                    "subject": "Verify Codesirius Account",
+                    "html": generate(username, user_id, verification_code),
+                },
+            )
+        except Exception as e:
+            self.logger.error(f"Failed to send email: {e}")
+            return None
+
+    def process_message(self, message: dict):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_message`

Add return type annotation: `None`

(ANN201)